doc/api/child_process.markdown

```diff
@@ -969,21 +1002,42 @@ then this will be `undefined`.
 `child.stdout` is an alias for `child.stdio[1]`. Both properties will refer
 to the same value.
 
-[`popen(3)`]: http://linux.die.net/man/3/popen
-[`ChildProcess`]: #child_process_child_process
+[`'error'`]: #child_process_event_error
+[`'exit'`]: #child_process_event_exit
+[`'message'`]: #child_process_event_message
+[`child.connected`]: #child_process_child_connected
+[`child.disconnect()`]: #child_process_child_disconnect
+[`child.kill()`]: #child_process_child_kill_signal
+[`child.send()`]: #child_process_child_send_message_sendhandle_callback
+[`child.stderr`]: #child_process_child_stderr
+[`child.stdin`]: #child_process_child_stdin
+[`child.stdout`]: #child_process_child_stdout
 [`child_process.exec()`]: #child_process_child_process_exec_command_options_callback
 [`child_process.execFile()`]: #child_process_child_process_execfile_file_args_options_callback
+[`child_process.execFileSync()`]: #child_process_child_process_execfilesync_file_args_options
+[`child_process.execSync()`]: #child_process_child_process_execsync_command_options
 [`child_process.fork()`]: #child_process_child_process_fork_modulepath_args_options
 [`child_process.spawn()`]: #child_process_child_process_spawn_command_args_options
 [`child_process.spawnSync()`]: #child_process_child_process_spawnsync_command_args_options
-[`ChildProcess#kill()`]: #child_process_child_kill_signal
-[`ChildProcess#send()`]: #child_process_child_send_message_sendhandle_callback
+[`ChildProcess`]: #child_process_child_process
 [`Error`]: errors.html#errors_class_error
 [`EventEmitter`]: events.html#events_class_events_eventemitter
-[`EventEmitters`]: events.html#events_class_events_eventemitter
+[`exec(2)`]: http://linux.die.net/man/2/exec
```

Comment:
Linking man pages is inconsistent with the rest of the node docs, and is being dealt with https://github.com/nodejs/node/pull/5073, and this will conflict with that attempt.